Patch-Source: https://github.com/andrewchambers/bupstash/pull/380
--
From 51efb583a98c6370fa5edbc55f64e43508ffbf8a Mon Sep 17 00:00:00 2001
From: Jakub Jirutka <jakub@jirutka.cz>
Date: Fri, 14 Apr 2023 22:40:56 +0200
Subject: [PATCH] Disable unused/unnecessary regex features to reduce binary
 size.

This reduces the size of the bupstash binary by ~400 kiB.
---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 4833806..7284227 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -41,7 +41,7 @@ humantime = "2.0.1"
 atty = "0.2"
 once_cell = "1.4"
 tar = "0.4"
-regex = "1"
+regex = { version = "1", default-features = false, features = ["std"] }
 globset = "0.4.8"
 chrono = { version = "0.4", features = ["serde"]}
 cfg-if = "0.1"
